### ISSUE TITLE
Can't create DirectLine when WebSocket not already in global scope

### ISSUE THREAD

**Issue by tobybrad** [Microsoft/BotFramework-DirectLineJS#5]
Minor issue instantiating an instance of DirectLine:

```
tobybrad@MININT-IGQ5M5R:~/src/alexa-bridge$ cat test.js

var directLine = require('botframework-directlinejs');
var connector = new directLine.DirectLine({secret:"topsecret"});

tobybrad@MININT-IGQ5M5R:~/src/alexa-bridge$ node test.js
/mnt/c/src/alexa-bridge/node_modules/botframework-directlinejs/built/directLine.js:69
        this.activity$ = this.webSocket && WebSocket !== undefined
                                           ^

ReferenceError: WebSocket is not defined
    at new DirectLine (/mnt/c/src/alexa-bridge/node_modules/botframework-directlinejs/built/directLine.js:69:44)
...
```

You probably wanted:

(typeof WebSocket != 'undefined' && WebSocket)

PR forthcoming..

**Comment by billba** [comment]
Great, thanks.

**Comment by billba** [comment]
Fixed by #6
